seqr/views/apis/data_manager_api.py

```diff
@@ -395,6 +395,30 @@ def load_rna_seq_sample_data(request, sample_guid):
     return create_json_response({'success': True})
 
 
+@data_manager_required
+def load_phenotype_pri_data(request):
+    request_json = json.loads(request.body)
+
+    file_name = request_json['file']
+    ignore_extra_samples = request_json.get('ignoreExtraSamples', False)
+
+    logger.info(f'Loading phenotype prioritization data from {file_name}', request.user)
+    records, info, warnings = load_phenotype_pri(file_name, request.user, ignore_extra_samples)
```

Comment:
Agree. I was trying to implement `load_phenotype_pri` in the same way as `load_rna_seq_outlier` and `load_rna_seq_tpm`. So these functions share the same util functions in the `dataset_utils.py`. To have better modularity, I put `load_phenotype_pri` in the util file. Since it is implemented differently, I can move it back to the API file.